src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -605,50 +710,38 @@ public JPanel getSexPanel() {
 		if (sexPanel == null) {
 			sexPanel = new JPanel();
 			ButtonGroup group = new ButtonGroup();
-			radiom = new JRadioButton(MessageBundle.getMessage("angal.common.male.btn"));
-			JRadioButton radiof = new JRadioButton(MessageBundle.getMessage("angal.common.female.btn"));
-			radioa = new JRadioButton(MessageBundle.getMessage("angal.common.all.btn"));
-			radioa.setSelected(true);
-			group.add(radiom);
-			group.add(radiof);
-			group.add(radioa);
-			sexPanel.add(radioa);
-			sexPanel.add(radiom);
-			sexPanel.add(radiof);
+			radioMale = new JRadioButton(MessageBundle.getMessage("angal.common.male.btn"));
+			radioFemale = new JRadioButton(MessageBundle.getMessage("angal.common.female.btn"));
+			radioAllGender = new JRadioButton(MessageBundle.getMessage("angal.common.all.btn"));
+			radioAllGender.setSelected(true);
+			group.add(radioMale);
+			group.add(radioFemale);
+			group.add(radioAllGender);
+			sexPanel.add(radioAllGender);
+			sexPanel.add(radioMale);
+			sexPanel.add(radioFemale);
 		}
 		return sexPanel;
 	}
 
-	public JPanel getNewPatientPanel() {
-		if (newPatientPanel == null) {
-			newPatientPanel = new JPanel();
-			ButtonGroup groupNewPatient = new ButtonGroup();
-			radioNew= new JRadioButton(MessageBundle.getMessage("angal.opd.new.btn"));
-			JRadioButton radioRea= new JRadioButton(MessageBundle.getMessage("angal.opd.reattendance.btn"));
-			radioAll= new JRadioButton(MessageBundle.getMessage("angal.common.all.btn"));
-			radioAll.setSelected(true);
-			groupNewPatient.add(radioAll);
-			groupNewPatient.add(radioNew);
-			groupNewPatient.add(radioRea);
-			newPatientPanel.add(radioAll);
-			newPatientPanel.add(radioNew);
-			newPatientPanel.add(radioRea);
-		}
-		return newPatientPanel;
-	}
-
-	/**
-	 * This method initializes jSelectionWardPanel
-	 *
-	 * @return javax.swing.JPanel
-	 */
-	private JPanel getJSelectionWardPanel() {
-		if (jSelectionWardPanel == null) {
-			jSelectionWardPanel = new JPanel();
-			jSelectionWardPanel.setLayout(new BoxLayout(jSelectionWardPanel, BoxLayout.Y_AXIS));
-			jSelectionWardPanel.add(getWardBox(), null);
-		}
-		return jSelectionWardPanel;
+	public JPanel getPatientTypePanel() {
+		JPanel patientTypePanel = new JPanel();
+		JLabel patientTypeLabel = new JLabel(MessageBundle.getMessage("angal.opd.patienttype.label"));
+		
+		ButtonGroup groupNewPatient = new ButtonGroup();
+		radioNewAttendance = new JRadioButton(MessageBundle.getMessage("angal.opd.new.btn"));
+		radioReAttendance = new JRadioButton(MessageBundle.getMessage("angal.opd.reattendance.btn"));
+		radioAllPatiens = new JRadioButton(MessageBundle.getMessage("angal.common.all.btn"));
```

Comment:
FWIW, the properties that end in `.btn` are really for buttons in button panels like `Cancel`, `Save`, `Edit`, etc. and not for radio buttons.

Reply:
I found it this way in other parts, so I did the same, should we change that?

![image](https://user-images.githubusercontent.com/2938553/215557079-af00cadb-7ac7-4b79-af51-50ade2f4fefa.png)

Reply:
At some point maybe but let's ignore it for now 😃